SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,29 +1,36 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
-# How to port a native Unreal project to the Unreal GDK
-
-> This workflow is very subject to change. We're aware that it's not optimized, and one of our priorities is to improve iteration times.
+# How to port a native Unreal project to the SpatialOS GDK for Unreal
 
 ## Prerequisites
 
 <%(TOC)%>
 
 Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
```

Comment:
There's a problem here: The cloning steps above talk specifically about cloning into StarterProject. For this doc, we want you to clone into the project you want to port. Separating the starter docs into different pages might help with this, and we can say "only do the engine setup" etc, then show you how to/where to clone here.

cc: @ernopp @jared-improbable